scripts/golos-boot-sequence/accounts.json

```diff
@@ -0,0 +1,21 @@
+{
+    "delegators": [
```

Comment:
"delegator" is somebody, who delegate vesiting. it's better to name it "witness", as in code